node/src/BaseClient.ts

```diff
@@ -3416,6 +3417,33 @@ export class BaseClient {
         );
     }
 
+    /**
+     * Returns the positions (longitude, latitude) of all the specified `members` of the
+     * geospatial index represented by the sorted set at `key`.
+     *
+     * See https://valkey.io/commands/geopos for more details.
+     *
+     * @param key - The key of the sorted set.
+     * @param members - The members for which to get the positions.
+     * @returns A 2D `array` which represents positions (longitude and latitude) corresponding
+     *     to the given members. If a member does not exist, its position will be `null`.
```

Comment:
```suggestion
     *     to the given members. The order of the returned positions matches the order of the input members. If a member does not exist, its position will be `null`.
```

Reply:
Can I add this update to python and java clients' docs too?